### ISSUE TITLE
Boolean condition operators should be placed below when clause

### ISSUE THREAD

**Issue by RalfKoban** [RalfKoban/MiKo-Analyzers#1058]
We should report a warning in case boolean condition operators are not placed below when clauses.

In addition, we should provide a codefix for that situation.

```c#
switch (o)
{
    case string s when s.Length > 0
        && s.Length < 10:
}
```
should be changed into
```c#
switch (o)
{
    case string s when s.Length > 0
                    && s.Length < 10:
}
```
